app/js/editQuestion.js

```diff
@@ -0,0 +1,108 @@
+let authorised = false;
+let user = getCookie('userEmail');
+
+getUser(user).then((user) => {
```

Comment:
isnt this in another JS file already? cant you just link that file rather than duplicating the code